test/unit/http_test.lua

```diff
@@ -0,0 +1,226 @@
+local json = require('json')
+local http_client = require('http.client')
+local metrics = require('metrics')
+
+local t = require('luatest')
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.role = require('roles.metrics-export')
+end)
+
+g.before_each(function(cg)
+    cg.counter = metrics.counter('some_counter')
+    cg.counter:inc(1, {label = 'ANY'})
+end)
+
+g.after_each(function(cg)
+    cg.role.stop()
+    metrics.registry:unregister(cg.counter)
+end)
+
+local function assert_none(uri)
+    local response = http_client.get(uri)
+    t.assert_not_equals(response.status, 200)
+    t.assert_not(response.body)
+end
+
+local function assert_json(uri)
+    local response = http_client.get(uri)
+    t.assert(response.body)
+
+    local data = response.body
+    local decoded = json.decode(data)
+    for _, node in ipairs(decoded) do
+        node.timestamp = nil
+    end
+    t.assert_equals(decoded, {
```

Comment:
To be honest, I was surprised that it works fine, since, after embedding metrics to the core binary, they are also enabled in `box.cfg` by default. It seems the reason is that `luatest` never calls `box.cfg` and runs `tarantool` just as a Lua interpreter, at least 1.0.1 one. Can we rely on that? It's a tricky question. As far as I remember latest discussions, user unit tests should not rely on possible luatest setup. Does it include our case? I think yes, since if luatest will decide to change this and start with `box.cfg` or from some configuration, these tests will start to fail.

To make things more carefully, we either need to
- use "covers/includes" approach rather than "equal";
- run these unit tests on a distinct server which has metrics disabled and default global labels unset;
- assert that `type(box.cfg) == 'function'`, so if it will ever change, we'll find out the reason rather fast;
- explicitly disable metrics on luatest instance with `box.cfg` call (not recommended).

Reply:
> why `luatest` should call something like box.cfg
Well, it's not likely that it should, but it definitely may and we can't forbid it to do so.

Reply:
> it definitely may and we can't forbid it to do so.

It may other dangerous things, so let's better wait for the real problem.